templates/user/dashboard/feeds.tmpl

```diff
@@ -63,12 +63,12 @@
 					{{else if eq .GetOpType 7}}
 						<span class="text truncate issue title has-emoji">{{index .GetIssueInfos 1}}</span>
 					{{else if eq .GetOpType 10}}
-						<span class="text truncate issue title has-emoji">{{.GetIssueTitle}}</span>
+						<a href="{{.GetCommentLink}}" class="text truncate issue title has-emoji">{{.GetIssueTitle}}</a>
 						<p class="text light grey has-emoji">{{index .GetIssueInfos 1}}</p>
 					{{else if eq .GetOpType 11}}
 						<p class="text light grey has-emoji">{{index .GetIssueInfos 1}}</p>
 					{{else if (or (or (eq .GetOpType 12) (eq .GetOpType 13)) (or (eq .GetOpType 14) (eq .GetOpType 15)))}}
-						<span class="text truncate issue title has-emoji">{{.GetIssueTitle}}</span>
+						<a href="{{.GetCommentLink}}"  class="text truncate issue title has-emoji">{{.GetIssueTitle}}</a>
```

Comment:
I think that the best solution is that I leave the links in place (except the link you mentioned). The .GetCommentLink function is also needed for backward-compatibility reasons. So actions that created before the update won't have a comment_id assigned. So .Comment.HTMLURL would not work. 
I think that adding these links does not need to be holed off to another PR.